drivers/entropy/Kconfig

```diff
@@ -6,19 +6,27 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+config ENTROPY_MUST_USE_OUT_OF_TREE_DRIVER
+  bool
+  help
+	  This option can be enabled by an out-of-tree library if it
+	  needs to take control over the entropy driver.
```

Comment:
Weird formatting.

Reply:
What is weird? Tab + spaces?

Reply:
Two spaces before `bool` and `help` instead of a tab. Makes it render wonky on GitHub too.